x-pack/plugins/observability_solution/observability_ai_assistant/server/service/client/adapters/bedrock/bedrock_claude_adapter.test.ts

```diff
@@ -217,16 +195,7 @@ describe('createBedrockClaudeAdapter', () => {
         ];
 
         expect(last(callSubActionFactory({ messages }).messages)!.content).toContain(
-          dedent(`<function_results>
-            <result>
-              <tool_name>my_tool</tool_name>
-              <stdout>
-                <myResponse>
-<myParam>myValue</myParam>
-</myResponse>
-              </stdout>
-            </result>
-          </function_results>`)
+          `{\"type\":\"tool_result\",\"tool\":\"my_tool\",\"myResponse\":{\"myParam\":\"myValue\"}}`
```

Comment:
nit: I think you can make the output more readable by `JSON.parse`'ing it first

```ts
        expect(JSON.parse(last(callSubActionFactory({ messages }).messages)!.content)).toEqual({
          myResponse: { myParam: 'myValue' },
          tool: 'my_tool',
          type: 'tool_result',
        });
```

Reply:
I'm actually using `toContain` here, so I can't use `toEqual`. But I can call JSON.stringify() instead of inline JSON, will make that change.

Reply:
done!

Reply:
Not a big deal but `toEqual` seems to work fine on my end. If we need partial object matching there's also `expect.objectContaining`

<img width="1001" alt="image" src="https://github.com/elastic/kibana/assets/209966/a26c3b46-f331-4548-b4ec-0d69d6b1b85c">

Reply:
oh that makes sense, I think that's just me being lazy. The system message and function requests need `toContain`, the function response doesn't. Will change.